libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list.component.ts

```diff
@@ -35,7 +37,11 @@ export class WattNavListComponent {}
 
 @NgModule({
   declarations: [WattNavListComponent],
-  exports: [WattNavListComponent, WattNavListItemScam],
+  exports: [
+    WattNavListComponent,
+    WattNavListItemScam,
+    WattNavListExternalItemScam,
```

Comment:
issue: `WattNavListItemComponent` is also exported in the `shell/nav-list/index.ts` barrel file. `WattNavListExternalItemComponent` should also be exported there for consistency.